tests/test_exceptions.py

```diff
@@ -248,6 +248,11 @@ def pycatch(exctype, f, *args):  # noqa: ARG001
     assert str(excinfo.value) == "this is a helper-defined translated exception"
 
 
+# TODO: Investigate this crash, see pybind/pybind11#5062 for background
+@pytest.mark.skipif(
+    sys.platform.startswith("win32") and "Clang" in pybind11_tests.compiler_info,
+    reason="Started sefaulting March 2024",
```

Comment:
If you want to keep this: there is a typo, and more correct is February:

```
"Started segfaulting February 2024"
```

Reply:
```suggestion
    reason="Started segfaulting February 2024",
```